<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formazione</class>
 <widget class="QDialog" name="Formazione">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inserisci formazione</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>198</y>
     <width>81</width>
     <height>63</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textFormazione">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>171</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPortieri">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboDifensori">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboCentrocampisti">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboAttaccanti">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDi">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Difensori</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCe">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Centrocampisti</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAt">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Attacanti</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Portieri</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>115</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Scegli giornata:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="giornataCombo">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBoxModulo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPortiere">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDifensoreDx">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDifensoreCx1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDifensoreCx2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDifensoreCx3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDifensoreSx">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCentrocampistaSx">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>470</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCentrocampistaCx3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>470</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCentrocampistaDx">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCentrocampistaCx2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCentrocampistaCx1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAttaccanteCx">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>530</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAttaccanteDx">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>530</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAttaccanteSx">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>530</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Formazione</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Formazione</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboPortieri</sender>
   <signal>activated(QString)</signal>
   <receiver>textFormazione</receiver>
   <slot>append(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboDifensori</sender>
   <signal>activated(QString)</signal>
   <receiver>textFormazione</receiver>
   <slot>append(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboCentrocampisti</sender>
   <signal>activated(QString)</signal>
   <receiver>textFormazione</receiver>
   <slot>append(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboAttaccanti</sender>
   <signal>activated(QString)</signal>
   <receiver>textFormazione</receiver>
   <slot>append(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
